cmd/snapctl/plugin.go

```diff
@@ -64,29 +64,19 @@ func loadPlugin(ctx *cli.Context) error {
 }
 
 func unloadPlugin(ctx *cli.Context) error {
-	pDetails := filepath.SplitList(ctx.Args().First())
-	var pType, pName string
-	var pVer int
-	var err error
+	pType := ctx.Args().Get(0)
+	pName := ctx.Args().Get(1)
+	pVer, err := strconv.Atoi(ctx.Args().Get(2))
```

Comment:
Would we need to check to verify that the number of args is correct here or does the library keep track of that for us?